webrtc.html

```diff
@@ -1840,14 +1840,18 @@ <h4>
                       that already has an existing {{RTCRtpTransceiver}} object,
                       <var>transceiver</var>, associated with it, as described in
                       <span data-jsep="applying-a-remote-desc">[[!RFC8829]]</span>,
-                      if the rid values specified in the simulcast attribute (up
-                      to the maximum number the implementation supports) differ
-                      in name, order or number from the ones previously used to
-                      populate
-                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}},
+                      if the first encoding in
+                      <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}
+                      [=map/contains=] a {{RTCRtpCodingParameters/rid}} member, and its
+                      value does not match the first rid in the simulcast attribute,
                       then [= description fails | fail =] the process of
                       applying <var>description</var>.
                       <div class="note">
+                        A change in the number of rid values is tolerated in
+                        remote offers as long as the first rid matches the rid
+                        of the first encoding that was previously negotiated.
```

Comment:
, or there are no rid values. (? - turning off simulcast via omitting the attribute could theoretically happen in subsequent offer/answer in a handoff situation)

Reply:
I'll test implementations and add that if it works.